src/gauss_pdf_and_cdf.py

```diff
@@ -0,0 +1,129 @@
+from os import makedirs
+from os.path import join
+import numpy as np
+import plotly.graph_objects as go
+from sympy import Symbol, pi, E, sqrt, integrate, oo
+from abc import ABC, abstractmethod
+from scipy import stats
+import math
+from config import FIGS_ROOT
+
+
+class ProbabilisticDistribution(ABC):
+
+    @abstractmethod
+    def __init__(self):
+        pass
+
+    @abstractmethod
+    def show_figure(self):
+        pass
+
+    @abstractmethod
+    def save_figure(self):
+        pass
+
+
+class Visualizer:
+
+    @staticmethod
+    def visualize(x, y, title, fig_processing):
```

Comment:
Add docstring for each function and class. See Google Style 3.8 Comments and Docstrings

Reply:
Add type annotation for each function and class. See Google Style 2.21 Type Annotated Code